.eslintrc.js

```diff
@@ -0,0 +1,208 @@
+// Copyright Amazon.com, Inc. or its affiliates. All Rights Reserved.
+// SPDX-License-Identifier: Apache-2.0
+
+const isCI = process.env.CI;
+
+module.exports = {
+  parser: '@typescript-eslint/parser',
+  extends: [
+    'eslint:recommended',
+    'plugin:@typescript-eslint/recommended',
+    'plugin:react/recommended',
+    'plugin:prettier/recommended',
+    'plugin:no-unsanitized/DOM',
+  ],
+  parserOptions: {
+    ecmaVersion: 2018,
+    sourceType: 'module',
+    ecmaFeatures: {
+      jsx: true,
+    },
+  },
+  plugins: ['unicorn', 'react-hooks', '@cloudscape-design/eslint-plugin', 'no-unsanitized', 'header'],
+  rules: {
+    '@typescript-eslint/no-empty-function': 'off',
+    '@typescript-eslint/no-namespace': 'off',
+    '@typescript-eslint/no-unused-vars': 'error',
+    '@typescript-eslint/consistent-type-definitions': ['error', 'interface'],
+    '@typescript-eslint/explicit-module-boundary-types': 'off',
+    '@typescript-eslint/no-explicit-any': 'off',
+    '@typescript-eslint/no-non-null-assertion': 'off',
+    'react/display-name': 'off',
+    'react/no-danger': 'error',
+    'react/no-unstable-nested-components': ['error', { allowAsProps: true }],
+    'react/prop-types': 'off',
+    'react/forbid-component-props': isCI ? 'off' : ['warn', { forbid: ['className', 'id'] }],
```

Comment:
This line is updated

Reply:
Why not to open a first quick rename-PR and then another one with updates?

Reply:
Good idea. Opened a new PR here for the renaming/format switching only: https://github.com/cloudscape-design/components/pull/1522